samples/system-test/buckets.test.js

```diff
@@ -86,6 +86,68 @@ it(`should set a bucket's default KMS key`, async () => {
   );
 });
 
+it(`should enable a bucket's Bucket Policy Only`, async () => {
+  // Remove the following setMetadata request after prod fix is released.
+  await bucket.setMetadata(
+    {defaultObjectAcl: null},
+    {predefinedDefaultObjectAcl: 'private'}
+  );
+
+  const results = await tools.runAsyncWithIO(
+    `${cmd} enable-bucket-policy-only ${bucketName}`,
+    cwd
+  );
+  assert.strictEqual(
+    (results.stdout + results.stderr).includes(
+      `Bucket Policy Only was enabled for ${bucketName}.`
+    ),
+    true
+  );
+  const metadata = await bucket.getMetadata();
+  assert.strictEqual(
+    metadata[0].iamConfiguration.bucketPolicyOnly.enabled,
+    true
+  );
+});
+
+it(`should get a bucket's Bucket Policy Only metadata`, async () => {
+  const results = await tools.runAsyncWithIO(
+    `${cmd} get-bucket-policy-only ${bucketName}`,
+    cwd
+  );
+
+  assert.strictEqual(
+    (results.stdout + results.stderr).includes(
+      `Bucket Policy Only is enabled for ${bucketName}.`
+    ),
+    true
+  );
+  const [metadata] = await bucket.getMetadata();
+  assert.strictEqual(metadata.iamConfiguration.bucketPolicyOnly.enabled, true);
```

Comment:
(style)
`assert.ok(metadata.iamConfiguration.bucketPolicyOnly.enabled);`

Reply:
Nice! I'll fix that, thanks @kinwa91!